src/NuGet/Microsoft.Orleans.OrleansTelemetryConsumers.Counters.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansTelemetryConsumers.Counters</id>
```

Comment:
I can do that. I just don't like the idea of have the counter control referenced in the project like we have today.

There is a tools folder on nuget that we can add it. What do you think?